tf/edgeml/trainer/bonsaiTrainer.py

```diff
@@ -1,18 +1,15 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT license.
-
 from __future__ import print_function
 import tensorflow as tf
 import edgeml.utils as utils
 import numpy as np
 import os
 import sys
-
+from sklearn.metrics import r2_score , mean_squared_error , mean_absolute_error
```

Comment:
`from sklearn.metrics import r2_score, mean_squared_error, mean_absolute_error`

Please remove the extra spaces.